documentation/API_documentation/home_devices_qod_readiness_checklist.md

```diff
@@ -0,0 +1,13 @@
+# HomeDevicesQoD API Readiness minimum criteria checklist
+
```

Comment:
Would the link to the template https://github.com/camaraproject/WorkingGroups/blob/main/Commonalities/documentation/API-Readiness-Checklist.md be helpful for the reader?

Reply:
Resolved in https://github.com/camaraproject/HomeDevicesQoD/pull/40/commits/27cb83819023aa630814b1a54a17955588d4c694